figures/settings/lms_production.py

```diff
@@ -35,7 +40,7 @@ def update_celerybeat_schedule(celerybeat_schedule_settings, figures_env_tokens)
                 ),
             }
 
-    if figures_env_tokens.get('ENABLE_DAILY_MAU_COLLECTION', True):
+    if figures_env_tokens.get('ENABLE_DAILY_MAU_IMPORT', False):
```

Comment:
renamed to use "IMPORT" instead of "COLLECTION" to be consistent with existing "ENABLE_DAILY_METRICS_IMPORT". Changing the names of this vars is up for discussion to update at a future time when we have to overhaul settings